### ISSUE TITLE
BHG does too much

### ISSUE THREAD

**Issue by icydee** [plainblack/Lacuna-Server-Open#427]
Can we redistribute some of the abilities of the BHG between other buildings?

E.g. could planet size (own planet) be done by some sort of research, or changes to environment?

**Comment by lemming552** [comment]
Agreed.  This is making me think of something that could work...

**Comment by Mr-Lizard** [comment]
yeah, but if you rearrange things...its gonna piss off those who invested
in BHGs.

**Comment by lemming552** [comment]
It's for the reboot.  No one will be starting with a BHG.
